data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/PstSetpointImpl.java

```diff
@@ -47,6 +47,10 @@ public Set<NetworkElement> getNetworkElements() {
         return Collections.singleton(networkElement);
     }
 
+    private int getNormalizedSetpoint(PhaseTapChanger phaseTapChanger) {
```

Comment:
I did like that because in both "client methods" we need also to retrieve the phaseTapChanger, so it enables to not look for the PST twice in the same method. But if you think it is clearer the way you proposed and that in terms of performance it will have little impact, I'll change.

Reply:
true !